app/policies/cdr/cdr_policy.rb

```diff
@@ -4,7 +4,7 @@ module Cdr
   class CdrPolicy < ::RolePolicy
     section 'Cdr/Cdr'
     self.allowed_actions += %i[allow_full_dst_number recording pcap]
-    alias_rule :dump?, :routing_simulation?, to: :perform?
+    alias_rule :routing_simulation?, to: :perform?
```

Comment:
I think **routing_simulation** should depends on access to **RoutingSimulation** page because this button just redirect user to other page